rca/schools/models.py

```diff
@@ -237,44 +334,44 @@ def page_nav(self):
             {"title": "Contact"},
         ]
 
-    def format_page_teasers(self, obj):
-        if not obj:
+    def get_related_projects(self):
+        """
+        Displays latest projects related to this school page.
+        Returns:
+            List of:
+                filtered and formatted manually related ProjectPages
+                or automatically fetch project pages from project > school relationship
+        """
+        from rca.projects.models import ProjectPage
+
+        manual_related_projects = [
+            i.page.id for i in self.manually_related_project_pages.all()
```

Comment:
I think this could help reduce queries `i.page.id for i in self.manually_related_project_pages.select_related('page')`

Reply:
:+1: added here https://github.com/torchbox/rca-wagtail-2019/pull/510/files#diff-6642bbb1b7771cb3e228f87b1a3d0e995ef2455634dd14f17d8076890956a875R348